README.md

```diff
@@ -129,14 +133,102 @@ axios.request(options).then(function (response) {
 });
 ```
 
-<!-- TODO USAGE EXAMPLES 
+<!-- TODO USAGE EXAMPLES  -->
 ## Usage
 
-Use this space to show useful examples of how a project can be used. Additional screenshots, code examples and demos work well in this space. You may also link to more resources.
+Our application is a backend application. We recommend using Postman to hit our routes.
 
-_For more examples, please refer to the [Documentation](https://example.com)_
+### Heroku Deployment
 
--->
+[Heroku Deployment](https://follow-the-nft-prod.herokuapp.com/)
+
+## Public Routes
+
+### Get: NFT by Token and Address
+
+```
+http://localhost:3001/<address>/<token_id>
+
+This will return back a NFT 
+```
+
+### Get: NFT by Address
+
+```
+http://localhost:3001/address/<address>
+
+Will return back wallet address 
+
+http://localhost:3001/addresses/<addresses>
+
+Will return back wallet addresses by how many address you add
+```
+
+### Get: Collection by Slug
+
+```
+http://localhost:3001/collection/<slug>
+
+Will return back a collection based on the slug passed in
+```
+
+## Auth Routes
+
+### Post: Register your account
+
+```
+http://localhost:3001/register
+
+Pass in your desired `username`, `password`, and optional `role` as `user` or `admin` in a json format.
+
+ex.
+
+{
+  "username": "testUserName",
+  "password": "testPassword",
+}
+
+Optional adding "role": "user"
```

Comment:
No role for admin.

Reply:
What do you mean?

Reply:
Oh you mean, add the role admin cause its not there

Reply:
- [x] added "role": "admin" example